Shops/ShopAngler.cs

```diff
@@ -124,24 +192,16 @@ public void InitShop(string currentShop)
                     }
                     break;
                 default:
-                    shopFishingPole(shop, ref nextSlot);
                     shop.item[nextSlot++].SetDefaults(ItemID.AnglerHat);
                     shop.item[nextSlot++].SetDefaults(ItemID.AnglerVest);
                     shop.item[nextSlot++].SetDefaults(ItemID.AnglerPants);
                     shop.item[nextSlot].SetDefaults(ItemID.HighTestFishingLine);
-                    shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     shop.item[nextSlot].SetDefaults(ItemID.AnglerEarring);
-                    shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     shop.item[nextSlot].SetDefaults(ItemID.TackleBox);
-                    shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     shop.item[nextSlot].SetDefaults(ItemID.FishermansGuide);
-                    shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     shop.item[nextSlot].SetDefaults(ItemID.WeatherRadio);
-                    shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     shop.item[nextSlot].SetDefaults(ItemID.Sextant);
-                    shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     shop.item[nextSlot].SetDefaults(ItemID.FishFinder);
```

Comment:
You need `++` otherwise everything will be merged into one slot.

Removing the price will make the accessories use their default prices, perhaps you should test out what their default prices are before removing their custom ones.